local-proxy/src/server.ts

```diff
@@ -168,8 +171,15 @@ router.post('/invoke', async (ctx) => {
       }
       const mod = require(joinedDir);
       fn = mod[handler];
+      // Cast to any so typescript doesn't complain against accessing __shiftjs__ on a function
+      if (!(typeof fn === 'function' && (fn as any).__shiftjs__ && (fn as any).__shiftjs__.exposed)) {
```

Comment:
Possible to extract as a common function between `server-function` and `local-proxy`.